bundles/org.openhab.binding.squeezebox/README.md

```diff
@@ -128,7 +138,7 @@ Currently, only favorites from the root level of the LMS favorites list are expo
 -   Add some favorites to your favorites list in LMS (local music playlists, Pandora, Slacker, Internet radio, etc.).
 Keep all favorites at the root level (i.e. favorites in sub-folders will be ignored).
 
--   If you're on an older openHAB build, you may need to delete and readd your squeezebox server and player things to pick up the new channels.
+-   If you're on an older openHAB build, you may need to delete and read your squeezebox server and player things to pick up the new channels.
```

Comment:
While it looks funny, readd is correct. To make more clear, you could hyphenate (i.e. re-add) as an alternative.

Reply:
Hi, I agree re-add is better.

Not sure how to change that, this is my first try at a pull request.